pages/community/precice-workshop-2022.md

```diff
@@ -0,0 +1,72 @@
+
+Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.
+
+If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).
+
+## Program
+
+The preliminary program reads as follows (more details will follow soon):
+
+* Monday, 21: preCICE introduction and preCICE introductory course
+* Tuesday, 22: user talks with live Q&A sessions (have a look at the [call for contributions](https://precice.org/precice-workshop-2022.html#call-for-contributions))
+* Wednesday, 23: developer talks with live Q&A sessions
+* Thursday, 24: interactive hands-on user support
+
+## Registration
+
+This is an academic, non-profit conference. However, we still have to collect a small registration fee.
+For early registrations (until January 17), the registration fee is
+
+* 50€ for members of universities and public research institutes and
+* 150€ for all other participants.
+
+Registration is still possible after this date, with a registration fee of
+
+* 100€ for members of universities and public research institutes and
+* 300€ for all other participants.
+
+Please use the [registration form](https://tagung.informatik-forum.org/preCICE2022/register) to register.
+
+For those who do not have funding to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.
+
+## Call for contributions
+
+Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory.
+
+Possible contributions include (non-exclusively):
+
+* New adapters for community codes
+* Coupling numerics
+* Comparison of different mapping, coupling, and communication methods
+* Coupling boundary conditions
+* Multi-scale coupling
+* Building, packaging, testing, documentation and other Research Software Engineering topics
+
+If you would like to submit a talk, please fill in the respective details during the registration process. We encourage all speakers to pre-record their talk (similar to the talks from [our last workshop](https://www.youtube.com/playlist?list=PLM2obW2udWdoxe1TGVuJXyIRM2JA8pOEL)) and submit them a few days before the workshop starts, since pre-recorded talks have worked out well in the past. However, a live presentation is still possible and the decision is up to each speaker. Have a look at the [important dates](https://precice.org/precice-workshop-2022.html#important-dates) below for relevant deadlines.
```

Comment:
@IshaanDesai @MakisH @uekerman what about this version?

Reply:
This is very clear :+1: good to go from my side!

Reply:
We still need a fixed deadline for the "a few days before the workshop". If it is not on my calendar, there is no deadline, and I will postpone it until it is too late. :wink: 

I think we should still have a default with an exception, instead of offloading the decision between the two to the speakers. See previous discussion.

Reply:
> We still need a fixed deadline for the "a few days before the workshop"

There is an exact deadline in the important dates section as described there.

Reply:
I know, but I felt it was a bit easy to miss with this formulation. But it does not matter anyway, since one needs to look at the important dates anyway.